nautobot/project-static/css/base.css

```diff
@@ -703,3 +703,24 @@ td span.hover_copy:hover .hover_copy_button {
 .navbar-inverse .navbar-nav > .dropdown > .dropdown-menu > li > a:hover {
     color: #ffffff;
 }
+
+/* Collapse icon on homepage panels */
+.collapse-icon {
+    float: right;
+    cursor: pointer !important;
+}
+
+/* Draggable homepage panels fade in effect on page load */
+#draggable-homepage-panels {
+    opacity: 0;
```

Comment:
This means these elements will start as and remain invisible if JS is disabled or if there's an error in loading `homepage_layout.js`, right? Not saying that's a 100% no-go but it does concern me.

Reply:
Those are good points.  The reason for this is since JS is loaded client side, the panels load in their initial state, then immediately "pop" into the custom ordering (similar to that dark mode flash of white a while ago).

This and the fade in hide the panels briefly to mask the popping effect.  I will need to think on another way to handle this.  Any ideas?

Reply:
I found a way using <noscript> tags on the homepage.  It is working well in my local testing.  Does this seem like a viable solution?